www/src/main/java/org/wso2/integration/ballerina/utils/Utils.java

```diff
@@ -21,20 +21,30 @@
 import org.apache.commons.io.FileUtils;
 import org.apache.commons.io.IOUtils;
 
+import java.io.BufferedReader;
 import java.io.File;
 import java.io.FileInputStream;
 import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
 import java.nio.charset.StandardCharsets;
 import java.util.Objects;
 import java.util.logging.Logger;
 
 import static org.wso2.integration.ballerina.constants.Constants.BALLERINA_CODE_MD_SYNTAX;
 import static org.wso2.integration.ballerina.constants.Constants.CODE;
 import static org.wso2.integration.ballerina.constants.Constants.CODE_MD_SYNTAX;
+import static org.wso2.integration.ballerina.constants.Constants.COMMIT_HASH;
 import static org.wso2.integration.ballerina.constants.Constants.EMPTY_STRING;
+import static org.wso2.integration.ballerina.constants.Constants.EQUAL;
+import static org.wso2.integration.ballerina.constants.Constants.FRONT_MATTER_SIGN;
+import static org.wso2.integration.ballerina.constants.Constants.GIT_COMMIT_ID;
+import static org.wso2.integration.ballerina.constants.Constants.HASH;
 import static org.wso2.integration.ballerina.constants.Constants.JAVA_CODE_MD_SYNTAX;
 import static org.wso2.integration.ballerina.constants.Constants.LICENCE_LAST_LINE;
 import static org.wso2.integration.ballerina.constants.Constants.NEW_LINE;
+import static org.wso2.integration.ballerina.constants.Constants.NOTE;
+import static org.wso2.integration.ballerina.constants.Constants.TITLE;
```

Comment:
Needs a private constructor for this class.

Reply:
Fixed